apps/bash_shared/defines.sh

```diff
@@ -1,5 +1,14 @@
 unamestr=$(uname)
 if [[ "$unamestr" == 'Darwin' ]]; then
+   if ! command -v brew &>/dev/null ; then
+       ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+   fi
+   if ! [ "${BASH_VERSINFO}" == 5 ]; then
```

Comment:
could be lower than ;)

Reply:
looks like it needs to be atleast version 4. so I changed it to check if its 4 or greater